src/storage/mod.rs

```diff
@@ -945,23 +937,13 @@ impl<E: Engine> Storage<E> {
     ) -> Result<()> {
         let mut modifies = Vec::with_capacity(DATA_CFS.len());
         for cf in DATA_CFS {
-            // We enable memtable prefix bloom for CF_WRITE column family, for delete_range
-            // operation, RocksDB will add start key to the prefix bloom, and the start key
-            // will go through function prefix_extractor. In our case the prefix_extractor
-            // is FixedSuffixSliceTransform, which will trim the timestamp at the tail. If the
-            // length of start key is less than 8, we will encounter index out of range error.
```

Comment:
Why it's not a problem anymore?

Reply:
Because we thought `DeleteRange` as a kv pair tiger panic when then the key length is less than 8 bytes, but actually it does not, so we can safely remove the padding.